cardano-testnet/src/Testnet/Types.hs

```diff
@@ -91,15 +92,20 @@ instance MonoFunctor (KeyPair k) where
 deriving instance Show (KeyPair k)
 deriving instance Eq (KeyPair k)
 
+instance {-# OVERLAPPING #-} Show (Some KeyPair) where
```

Comment:
This is due to:
```
instance Show (Some f) where
  showsPrec _ (Some v) = showsTypeRep (typeOf v)
```
I don't think it's used anywhere so lets remove it to avoid having to use `OVERLAPPING`.

Reply:
I agree, I think it will be better to provide specific show instances e.g. `Show (Some Era` instead.

Reply:
Follow-up pr: https://github.com/IntersectMBO/cardano-api/pull/665